### PR TITLE
Wd-59 Changing static html to jinja2

### PR DESCRIPTION
**Pull request by Pacu2**
I've changed static html to jinja2, website is running way faster!

### CODE REVIEW

**Comment by Pacu2** [comment]
https://kindergarten.stxnext.pl/secure/RapidBoard.jspa?rapidView=11&view=detail&selectedIssue=WDP-59

**Comment by dpolcyn** [comment]
Please squash commits

File: src/presence_analyzer/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <div id="content">
+            <h2>{% block title %}{% endblock %}</h2>
+            <p>
+                <select id="user_id" style="display: none">
+                    <option value="">--</option>
+                </select>
+                <div id="chart_div" style="display: none">
+                </div>
+                <div id="loading">
+                    <img src="/static/img/loading.gif" />
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/base.html]: url_for

File: src/presence_analyzer/tests.py [hunk 2]
```diff
@@ -204,7 +213,7 @@ def test_group_by_start_end_time(self):
         data = utils.get_data()
 
         self.assertEqual(proper_data, utils.group_by_start_end_time(data[10]))
-
+        self.assertEqual(proper_data, utils.group_by_start_end_time(data[10]))
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: You have this same assert in 215 line

File: src/presence_analyzer/views.py [hunk 3]
```diff
@@ -84,3 +85,14 @@ def presence_start_end_time(user_id):
         (calendar.day_abbr[weekday], start, end)
         for weekday, (start, end) in enumerate(weekdays)
     ]
+
+
+@app.route('/<string:template_name>')
+def render_view(template_name):
+    """Render template based on html file"""
+    if template_name.endswith(".html"):
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: We use ' in python code not ".
Check if not endswith .html then add .html to template name, then you dont have to use .strip()

**Pull request closed without merging**